Sources/SafeDICore/Models/Scope.swift

```diff
@@ -78,7 +78,8 @@ final class Scope: Hashable {
 
     func createScopeGenerator(
         for property: Property? = nil,
-        propertyStack: OrderedSet<Property> = []
+        propertyStack: OrderedSet<Property> = [],
+        erasedToConcreteExistential: Bool = false
```

Comment:
we use a default value here (and in the other arguments) because root types call into this method, and root types are never erased.